packages/babel-plugin-transform-simplify-comparison-operators/__tests__/eqnull-test.js

```diff
@@ -0,0 +1,43 @@
+jest.autoMockOff();
+
+const babel = require("babel-core");
+const plugin = require("../src/index");
+
+function transform(code) {
+  return babel.transform(code, {
+    plugins: [plugin]
+  }).code;
+}
+
+describe("simplify-comparison-operators-plugin", () => {
+  it("should shorten `x == undefined`", () => {
+    const source = "x == undefined;";
+    const expected = "x == null;";
+    expect(transform(source)).toBe(expected);
+  });
+  it("should not shorten `x === undefined`", () => {
+    const source = "x === undefined;";
+    expect(transform(source)).toBe(source);
+  });
+
+  it("should shorten `undefined == null`", () => {
+    const source = "undefined == null";
```

Comment:
That can be fixed in `path.evaluate` of babel if that doesn't handle it. Or it's an ordering issue of who runs first - #61 #422.